plugins/BulkTracking/Tracker/Response.php

```diff
@@ -55,7 +60,8 @@ private function formatException(Tracker $tracker, Exception $e)
         // when doing bulk tracking we return JSON so the caller will know how many succeeded
         $result = array(
             'status'  => 'error',
-            'tracked' => $tracker->getCountOfLoggedRequests()
+            'tracked' => $tracker->getCountOfLoggedRequests(),
+            'skipped' => $this->skippedRequests,
```

Comment:
See above, maybe rather `invalid` or similar than `skipped`? I do not really mind though to leave it like this